src/main/java/htsjdk/samtools/seekablestream/ByteArraySeekableStream.java

```diff
@@ -27,7 +27,11 @@ public long position() throws IOException {
 
     @Override
     public void seek(long position) throws IOException {
-        this.position = position;
+        if (position < 0) {
+			throw new IOException("Negative seek offset");
+		} else {
+			this.position = position;
+		}
```

Comment:
This is very nitpicky, but could you use spaces instead tabs so that the formatting matches the rest of the file?